pkg/query/storeset.go

```diff
@@ -0,0 +1,258 @@
+package query
+
+import (
+	"context"
+	"math"
+	"sync"
+	"time"
+
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
Can we bring back the explicit naming of the imports as they package namespaces diverge from the import path suffix.

Reply:
yup